start-site/src/main/resources/application.yml

```diff
@@ -78,6 +78,16 @@ initializr:
             version: 3.0.0
           - compatibilityRange: "[3.1.0-M1, 3.3.0-M1)"
             version: 3.1.0
+      spring-ai:
+        groupId: org.springframework.ai
+        artifactId: spring-ai-bom
+        versionProperty: spring-ai.version
+        mappings:
+          - compatibilityRange: "[3.0.0,3.3.0-M1)"
+            # TODO: Replace with released version
+            version: 0.8.0-SNAPSHOT
+            # TODO: Remove once on Maven central
+            repositories: spring-snapshots
```

Comment:
Which hopefully will land on maven central.

Reply:
The first time for a project is always the worst! ;)

Reply:
It's not on Maven Central (https://repo.maven.apache.org/maven2/org/springframework/ai/spring-ai-bom/0.8.0/), but I found it on Spring Milestones (https://repo.spring.io/ui/native/milestone/org/springframework/ai/spring-ai-bom/0.8.0/). Is this on purpose?